src/components/Tool/MergeContacts/Contact.tsx

```diff
@@ -156,17 +150,31 @@ const Contact: React.FC<Props> = ({ contact1, contact2, update }) => {
                         width: '100%',
                       }}
                     >
-                      <Typography variant="h6">{contact1.title}</Typography>
+                      <Typography variant="h6">{contact1.name}</Typography>
                     </Box>
 
-                    <Typography>{t('Status:')}</Typography>
-                    <Typography>{contact1.address.street}</Typography>
-                    <Typography>{`${contact1.address.city}, ${contact1.address.state} ${contact1.address.zip}`}</Typography>
+                    <Typography>
+                      {t('Status: {{status}}', { where: contact1.status })}
+                    </Typography>
+                    {contact1.primaryAddress ? (
+                      <>
+                        <Typography>
+                          {contact1.primaryAddress.street}
+                        </Typography>
+                        <Typography>{`${contact1.primaryAddress.city}, ${contact1.primaryAddress.state} ${contact1.primaryAddress.postalCode}`}</Typography>
+                      </>
+                    ) : (
+                      ''
+                    )}
                     <Typography>
                       {t('From: {{where}}', { where: contact1.source })}
                     </Typography>
                     <Typography>
-                      {t('On: {{when}}', { when: contact1.date })}
+                      {t('On: {{when}}', {
+                        when: DateTime.fromISO(
```

Comment:
Same here

Reply:
The status one is wrong but doesn't this one match?